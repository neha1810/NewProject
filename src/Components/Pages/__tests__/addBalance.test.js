import React from 'react';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import AddBalance from '../addBalance/addBalance';

configure({ adapter: new Adapter() });

describe('AddBalance', () => {
    let wrapper, wrapper1, wrapper2;
    beforeEach(() => {
        wrapper = shallow(
            <AddBalance />
        );
        wrapper1 = shallow(<AddBalance />)
        wrapper2 = shallow(<AddBalance />)
    })


    it('render the AddBalance component', () => {
        expect(wrapper).toMatchSnapshot();
    });
    it('input', () => {
        expect(wrapper.find('amount')).toMatchSnapshot();
        expect(wrapper.find('.a-button-input')).toMatchSnapshot();
        expect(wrapper1.find('.a-button-input')).toMatchSnapshot();
        expect(wrapper2.find('.a-button-input')).toMatchSnapshot();
    });

})