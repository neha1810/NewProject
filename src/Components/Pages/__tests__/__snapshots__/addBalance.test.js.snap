// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddBalance input 1`] = `ShallowWrapper {}`;

exports[`AddBalance input 2`] = `ShallowWrapper {}`;

exports[`AddBalance input 3`] = `ShallowWrapper {}`;

exports[`AddBalance input 4`] = `ShallowWrapper {}`;

exports[`AddBalance render the AddBalance component 1`] = `ShallowWrapper {}`;
